runtime/errors.rs

```diff
@@ -153,12 +154,25 @@ pub fn get_nix_error_class(error: &nix::Error) -> &'static str {
   }
 }
 
+pub fn get_cache_error(error: &CacheError) -> &'static str {
```

Comment:
move this to deno_cache? similar to https://github.com/denoland/deno/pull/26111/files#r1795289078